api/envoy/config/core/v3/protocol.proto

```diff
@@ -94,6 +94,12 @@ message QuicProtocolOptions {
   // Probes the peer at the configured interval to solicit traffic, i.e. ACK or PATH_RESPONSE, from the peer to push back connection idle timeout.
   // If absent, use the default keepalive behavior of which a client connection sends PINGs every 15s, and a server connection doesn't do anything.
   QuicKeepAliveSettings connection_keepalive = 5;
+
+  // An array of QUIC connection options represented in a serialized string, separated by commas.
```

Comment:
nits: A comma-separated list of strings representing QUIC connection options defined in QUICHE  and to be sent by upstream connections.
Can you reference to the connection options definition file in QUICHE?